test/unit/lib/country-data.test.js

```diff
@@ -0,0 +1,62 @@
+const {
+    countryOptions,
+    overrideOptions,
+    commonOptions,
+    processedCountryOptions,
+    subdivisionOptions
+} = require('../../../src/lib/country-data');
+
+describe('unit test lib/country-data.js', () => {
+    test('countryOptions', () => {
+        expect(typeof countryOptions).toBe('object');
+        expect(countryOptions.length).toEqual(249);
+        expect(countryOptions[0]).toEqual({value: 'af', label: 'Afghanistan'});
+    });
+
+    test('overrideOptions', () => {
+        expect(typeof overrideOptions).toBe('function');
+        const overriddenCountryOptions = overrideOptions({
+            ca: 'Canadia',
+            gb: 'The United Kingdom of Great Britain and Northern Ireland'
+        });
+        expect(overriddenCountryOptions.length).toEqual(249);
+        expect(countryOptions[38]).toEqual({value: 'ca', label: 'Canada'});
+        expect(overriddenCountryOptions[38]).toEqual({value: 'ca', label: 'Canadia'});
+        expect(countryOptions[232]).toEqual({value: 'gb', label: 'United Kingdom'});
+        expect(overriddenCountryOptions[232]).toEqual({
+            value: 'gb',
+            label: 'The United Kingdom of Great Britain and Northern Ireland'
+        });
+    });
+
+    test('commonOptions', () => {
```

Comment:
Might be helpful to have messages on some of these expects to explain what you're testing. 
I know you're trying to get CA and GB to show up at the top and in alphabetical order, but that isn't apparent just from reading the code.

Reply:
will do

Reply:
ok, i clarified these a lot. take a look!